job-server/src/spark.jobserver/io/JobDAO.scala

```diff
@@ -68,7 +68,7 @@ trait JobDAO {
    *
    * @return
    */
-  def getJobInfos(limit: Int): Future[Seq[JobInfo]]
+  def getJobInfos(limit: Int,status:String = ""): Future[Seq[JobInfo]]
```

Comment:
I think it is way better to pass `Option[String]` as status, then it is very clear when we do not want to filter by some status